ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
"If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles." [1].

Meaning, we need to break the ``etcd`` role out into ``etcd-start``, ``etcd-configure``, ``etcd-restart`` and other roles if we want to support more scenarios than just an ``etcd-setup``. 

@stephenrlouie Is this, what you meant when you were talking about "The end state is to deploy each role like a 'micro-service' that has granular control so one can spin up, spin down, individual roles without having damaging effects across the cluster." ?

[1] http://docs.ansible.com/ansible/playbooks_roles.html#roles